optimism/src/keccak/lookups.rs

```diff
@@ -42,7 +42,8 @@ impl<Fp: Field> Lookups for KeccakEnv<Fp> {
             // Power of two corresponds to 2^pad_length
             // Pad suffixes correspond to 10*1 rule
             // Note: When FlagLength=0, TwoToPad=1, and all PadSuffix=0
-            self.add_lookup(Lookup::read_one(
+            self.add_lookup(Lookup::new(
```

Comment:
`Lookup::new` is barely less verbose than the record. It feels like it would be clearer to have it expanded and delete the helper, like
```rust
self.add_lookup(Lookup {
    mode: rw,
    magnitude: Self::Variable::one(),
    table_id: LookupTable::PadLookup,
    value: vec![...],
});
```

Reply:
absolutely, the sole reason that I changed `read_one` for `new(LookupMode::Read`, was that I thought `self.lookups(rw)` would be called from both `Write` and `Read` modes. Given that now I know modes should be fixed ahead of time and these ones will always be `Read`, it makes sense to revert this change. I will check everything I changed in this PR, because maybe I can just close it, or make it much smaller in scope.